src/features/content-explorer/item-list/ItemList.js

```diff
@@ -287,6 +293,7 @@ ItemList.propTypes = {
     width: PropTypes.number.isRequired,
     height: PropTypes.number.isRequired,
     rowHeight: PropTypes.number,
+    tableHeaderHeight: PropTypes.number,
```

Comment:
since we call the other prop just `rowHeight`, could we call this `headerHeight`?

Reply:
sure